src/Standards/Generic/Docs/Arrays/ArrayIndentStandard.xml

```diff
@@ -0,0 +1,101 @@
+<documentation title="Array Indent">
+    <standard>
+    <![CDATA[
+    The opening square bracket/parenthesis of a multi-line array must be indented at least to the same level of the start of the statement.
```

Comment:
For this `<standard>` block, I'm specifically mentioning multi-line arrays. It is the only error that could apply to a single-line array, but single-line arrays are not handled by this sniff. The example that I have in mind is:

```php
    $a = 
[1, 2];
```

Reply:
Just tested, that snippet would not trigger the sniff anyway, so we can forget about it.